src/components/ButtonsGroup/ButtonsGroup.tsx

```diff
@@ -1,41 +1,52 @@
-import React, { Children, useMemo, useState } from 'react';
-import { FlatList, StyleSheet, View } from 'react-native';
-import { sizes } from '@magnetis/astro-tokens';
+import React, { useState } from 'react';
+import { FlatList, Pressable, PressableProps, StyleSheet, Text } from 'react-native';
+import { sizes, colors, radius } from '@magnetis/astro-tokens';
 
-import ButtonContent, { ButtonContentProps } from './ButtonContent';
+type Item = {
+  value: string;
+  onPressItem: () => void;
+};
 
-import type { Callback } from './types';
-
-export interface ButtonsGroupProps {
-  testID?: string;
-  children: React.FunctionComponentElement<ButtonContentProps>[];
+export interface ButtonsGroupProps extends PressableProps {
+  items: Item[];
+  rounded?: boolean;
+  activeItemColor?: string;
```

Comment:
Acredito que seja melhor aqui definir as variações ao invés de permitir receber qualquer cor como string.
Minha sugestão é remover essa prop e colocar no lugar outras duas que estejam ligadas a variação.
```suggestion
  active?: boolean;
  inversed?: boolean;
```
Assim evitamos que percamos controle de como os componentes serão usados e limitamos somente ao que foi especificado no DS. No futuro, caso tenhamos necessidade de mais alguma variação, será inicializado pelo time de PDs para depois adicionarmos ao código.